src/Tabs.tsx

```diff
@@ -1 +1,41 @@
-// implement a component here
+import React from 'react';
+import './Tabs.scss';
+
+type Props = {
+  tabsList : Tab[];
+  selectedTabId : string;
+  onTabSelected : (tab : Tab) => void,
+};
+
+export const Tabs : React.FC<Props> = ({
+  tabsList,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <div className="container">
+      {tabsList.map((singleTab, index) => (
+        <div
+          key={singleTab.id}
+          className="tab"
+        >
+          <button
+            type="button"
+            className="tab__pick"
+            onClick={() => {
+              onTabSelected(singleTab);
+            }}
+          >
+            {`Tab ${index + 1}`}
```

Comment:
You can write here just singleTab.title, i guess. So you don't need index as parametr in map method.